dialogue-apache-hc4-client/src/main/java/com/palantir/dialogue/hc4/ApacheHttpClientChannels.java

```diff
@@ -145,21 +146,37 @@ private static MetricName clientPoolSizeMetricName(String clientName, String sta
 
     /** Intentionally opaque wrapper type - we don't want people using the inner Apache client directly. */
     public static final class CloseableClient implements Closeable {
+        private final String name;
         private final CloseableHttpClient client;
+        private final PoolingHttpClientConnectionManager pool;
         private final ResponseLeakDetector leakDetector;
 
         @Nullable
         private final ExecutorService executor;
 
         CloseableClient(
-                CloseableHttpClient client, ResponseLeakDetector leakDetector, @Nullable ExecutorService executor) {
+                String name,
+                CloseableHttpClient client,
+                PoolingHttpClientConnectionManager pool,
+                ResponseLeakDetector leakDetector,
+                @Nullable ExecutorService executor) {
+            log.debug("Apache client created", SafeArg.of("name", name));
```

Comment:
we already produce a meter information internally, not sure if this is necessary

Reply:
Metrics come out at a 30second granularity. Also the semantics of the metric produced by WC could change and we wouldn't be able to tell from the graph. This is as close to the 'ground truth' as I think we can get.

Reply:
I think info level is reasonable here

Reply:
ya we'd expect this to happen incredibly infrequently right?

Reply:
yep